livekit-plugins/livekit-plugins-browser/src/CMakeLists.txt

```diff
@@ -11,8 +11,19 @@ set(GLFW_INSTALL OFF CACHE BOOL "" FORCE)
 FetchContent_Declare(glfw GIT_REPOSITORY https://github.com/glfw/glfw.git GIT_TAG 3.4)
 FetchContent_MakeAvailable(glfw)
 
-FetchContent_Declare(imgui GIT_REPOSITORY https://github.com/ocornut/imgui GIT_TAG origin/docking)
+FetchContent_Declare(
+  imgui 
+  GIT_REPOSITORY https://github.com/ocornut/imgui 
+  GIT_TAG origin/docking
+  GIT_SHALLOW TRUE
+)
 FetchContent_GetProperties(imgui)
+FetchContent_Populate(imgui)
+execute_process(
+    COMMAND git fetch --unshallow
+    WORKING_DIRECTORY ${imgui_SOURCE_DIR}
+)
```

Comment:
Why do we want to unshallow?

Reply:
Thanks for asking about this! I tried shallow cloning ImGui and then unshallowing since the regular clone was timing out. This worked perfectly for me and resolved the timeout issue. Hope this helps! 😊

Reply:
Ah ok interesting, I didn't have this issue but my guess is that we could even skip the unshallow part since we don't need the history

Reply:
I applied the requested change :+1: